src/Rings/hilbert.jl

```diff
@@ -223,29 +180,27 @@ end
 function involves(t::PP, IndexList::Vector{Int64})
   # ASSUMES: indexes in IndexList are all in range
   for i in IndexList
-    @inbounds if t[i] != 0
-      return true
-    end
+    @inbounds t[i] != 0 && return true
   end
-  return  false
+  return false
 end
 
 
 function Base.show(io::IO, t::PP)
-  if all(t.expv .== 0)   # isone(PP)  # why doesn't this work ???
+  if !isone(t)
```

Comment:
```suggestion
  if isone(t)
```
as suggested by @JohnAAbbott